From 79bdfa05497411b54fd890624158272784e89fc9 Mon Sep 17 00:00:00 2001
From: Tobias Mayer <tobim@fastmail.fm>
Date: Mon, 14 Feb 2022 13:38:25 +0100
Subject: [PATCH] Add missing format string argument

---
 plugins/broker/src/reader.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/broker/src/reader.cpp b/plugins/broker/src/reader.cpp
index 2245964e35..507c3c1f89 100644
--- a/plugins/broker/src/reader.cpp
+++ b/plugins/broker/src/reader.cpp
@@ -162,7 +162,7 @@ caf::error reader::dispatch(const ::broker::data& msg, size_t max_slice_size,
                                                     "slice builder");
         } else {
           VAST_DEBUG("{} ignores identical layout for stream ID {}: {}", name(),
-                     stream_id);
+                     stream_id, layout);
         }
       } else {
         VAST_INFO("{} got schema for new stream {}", name(), stream_id);
